<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMdialog</class>
 <widget class="QDialog" name="FMdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matching Preview</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/imgs/sew1.png</normalon>
   </iconset>
  </property>
  <widget class="QLabel" name="pLbl_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>30</y>
     <width>225</width>
     <height>225</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="pLbl_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>305</y>
     <width>225</width>
     <height>225</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="pLbl_3">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>30</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>540</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="ptLbl_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>226</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="ptLbl_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>280</y>
     <width>226</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="ptLbl_3">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>5</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Hack</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>540</y>
     <width>319</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Hack</family>
       </font>
      </property>
      <property name="text">
       <string>Choose Reference Stack:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="pSubStackCB">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Hack</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>FMdialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
